### PR TITLE
fix more incorrect array nullability annotations

### PR DESCRIPTION
**Pull request by Machine-Maker**
Replaces https://github.com/PaperMC/Paper/pull/10013

### CODE REVIEW

File: paper-api/src/main/java/org/bukkit/World.java [hunk 1]
```diff
@@ -346,8 +346,7 @@ default boolean isChunkGenerated(long chunkKey) {
      *
      * @return Chunk[] containing all loaded chunks
      */
-    @NotNull
-    public Chunk[] getLoadedChunks();
+    public @NotNull Chunk @NotNull[] getLoadedChunks();
```
Comment (lynxplay) [inline paper-api/src/main/java/org/bukkit/World.java]: Formatting

File: paper-api/src/main/java/org/bukkit/inventory/view/EnchantmentView.java [hunk 2]
```diff
@@ -36,14 +37,13 @@ public interface EnchantmentView extends InventoryView {
      *
      * @return The enchantment offers that are provided
      */
-    @NotNull
-    EnchantmentOffer[] getOffers();
+    @NotNull EnchantmentOffer @Nullable [] getOffers();
```
Comment (lynxplay) [inline paper-api/src/main/java/org/bukkit/inventory/view/EnchantmentView.java]: Wrong way around.
The arrays are not null, its elements may be.
Same for below.

**Review by lynxplay — CHANGES_REQUESTED** [review]

**Comment by Lulu13022002** [comment]
Is that even relevant to do before the jspecify migration for not null types that are not wrong? there's also some formatting issues for World#getLoadedChunks, SimplexNoiseGenerator#dot

**Comment by Machine-Maker** [comment]
Yeah, should do it. The jspecify migration is just gonna happen in chunks when you modify a file, I don’t think there’s gonna be one “huge” PR to do it all at once. Reviewing it would be a nightmare

**Review by lynxplay — APPROVED** [review]

**Pull request merged**